dev-workflow-ui/src/ch/ivyteam/workflowui/tasks/TasksDataModel.java

```diff
@@ -78,7 +78,9 @@ protected void applyCustomFilter(TaskQuery taskQuery) {}
   private void checkIfShowAll(TaskQuery taskQuery) {
     if (!isShowAll()) {
       taskQuery.where().and(TaskQuery.create().where().currentUserIsInvolved());
-      taskQuery.where().and().not(TaskQuery.create().where().activatorId().isEqual(ISecurityContext.current().users().system().getSecurityMemberId()));
+      if (UserUtil.isAdmin()) {
```

Comment:
@ivy-cst the problem from what i understand is that if you start a casemap which has for example a dialog and then during the dialog you go to the homepage, you will see the case that you created but the task of the casemap/dialog is not visible because its filtered for some reason (but the admin "Developer" can see the task)